src/metabase/async/streaming_response.clj

```diff
@@ -79,7 +79,7 @@
   `async-context`."
   [^AsyncContext async-context f ^OutputStream os finished-chan canceled-chan]
   {:pre [(some? os)]}
-  (let [task (bound-fn []
+  (let [task (^:once fn* []
```

Comment:
Is there documentation anywhere about what `^:once` does? I can't find anything about it online.

Reply:
This is the only place I think it's really explained: https://groups.google.com/g/clojure/c/Ys3kEz5c_eE

the TL;DR is `^:once` is basically telling the compiler that you're guaranteeing the function will be ran once and only once which can result in it cleaning up closed-over variables right away instead of holding on to them

I don't think it's actually important here now that I look at it since it shouldn't make a difference for any of the things we're capturing

Reply:
IIRC `ONCEFN` can get inlined too so regardless of holding on to captured variables it can still result in more efficient code...  altho that wouldn't make a difference here either since we're executing it on a different thread anyway